refinery/ui/source/js/tool-launch/ctrls/tool-params-ctrl.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Tool Params Ctrl
+ * @namespace ToolParamsCtrl
+ * @desc Ctrl for rpToolParams directive, which is the parameter's panel.
+ * Include collapsing, displaying parameters, and form for parameters.
+ * @memberOf refineryApp.refineryToolLaunch
+ */
+(function () {
+  'use strict';
+
+  angular
+    .module('refineryToolLaunch')
+    .controller('ToolParamsCtrl', ToolParamsCtrl);
+
+  ToolParamsCtrl.$inject = [
+    '$scope',
+    'toolParamsService'
+  ];
+
+
+  function ToolParamsCtrl (
+    $scope,
+    toolParamsService
+  ) {
+    var paramsService = toolParamsService;
+    var vm = this;
+    vm.isToolParamsCollapsed = false; // tracks the parameters panel
```

Comment:
Thoughts on starting this collapsed? 
I think if people are really dying to edit a tools params they will, otherwise they'll probably run them with the defaults at first.